web3/manager.py

```diff
@@ -107,6 +110,9 @@ def request_blocking(self, method, params):
         if "error" in response:
             raise ValueError(response["error"])
 
+        if response['result'] is None:
+            raise ValueError(f"The call to {method} did not return a value.")
```

Comment:
@kclowes @dylanjw I'm not greatly familiar with the async gameplan for web3 - but it occurred to me that if we are raising exceptions for jsonrpc calls that return `None` in synchronous requests, then we should have the same behavior in the async requests.

Reply:
This looks good to me.